packages/experimental/Link/src/useLink.ts

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { useAsPressable, useKeyProps, useOnPressWithFocus, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+import { LinkProps, LinkState } from './Link.types';
+import { memoize } from '@fluentui-react-native/framework';
+import { AccessibilityState, Linking, Platform } from 'react-native';
+
+export const useLink = (props: LinkProps): LinkState => {
+  // attach the pressable state handlers
+  const defaultComponentRef = React.useRef(null);
+  const {
+    accessibilityRole,
+    onPress,
+    onAccessibilityTap,
+    tooltip,
+    url,
+    accessibilityState,
+    componentRef = defaultComponentRef,
+    disabled,
+    enableFocusRing,
+    focusable,
+    ...rest
+  } = props;
+  const isDisabled = !!disabled;
+
+  const [visitedState, setVisitedState] = React.useState({ visited: false });
+  const linkOnPress = React.useCallback(
+    (e) => {
+      if (!disabled) {
```

Comment:
Just curious - does the onPress() callback get fired on a disabled component?

Reply:
it shouldn't. I can remove this check since it's not really necessary